File: LayoutDemoOC/LayoutDemoOC/ViewController.m
//
//  ViewController.m
//  LayoutDemoOC
//
//  Created by Pluto Y on 16/2/29.
//  Copyright © 2016年 pluto-y. All rights reserved.
//

#import "ViewController.h"
#import "AutoresizingViewController.h"
#import "AutoLayoutViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
}

/**
 *  Autoresizing的demo
 */
- (IBAction)gotoAutoresizing:(id)sender {
    AutoresizingViewController *controller = [[AutoresizingViewController alloc] init];
    [self presentViewController:controller animated:YES completion:nil];
}

/**
 *  AutoLayout的Demo1
 *
 */
- (IBAction)gotoAutoLayout:(id)sender {
    AutoLayoutViewController *controller = [[AutoLayoutViewController alloc] init];
    [self presentViewController:controller animated:YES completion:nil];
}

@end

File: HealthKitTotorial/HealthKitTotorial/ViewController.swift
//
//  ViewController.swift
//  HealthKitTotorial
//
//  Created by ChipSea on 15/12/6.
//  Copyright © 2015年 pluto-y. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        self.title = "Health Kit Demo"
    }


    @IBAction func authorizeHealthKitClick(sender: AnyObject) {
        HealthKitUtil.authorizeHealthKit { (success, error) -> Void in
            if !success {
                print("Error : 获取授权失败, error : \(error.localizedDescription)")
            } else {
                print("获取授权成功")
            }
        }
    }
}

